### PR TITLE
Include ViewModel to Benchmarks

### PR DESCRIPTION
**Pull request by arggh**

### CODE REVIEW

**Comment by arggh** [comment]
ping @ManuelDeLeon , in case you want to check that the benchmark for ViewModel is implemented like it should be.

File: client/viewmodel/main.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import { Template } from 'meteor/templating';
+import { Tracker } from 'meteor/tracker';
+import { viewmodel } from 'meteor/manuel:viewmodel';
+
+import profile from '../profile';
+import collections from '../collections';
+
+import './main.html';
+
+ViewModel.mixin({
```
Comment (mitar) [inline client/viewmodel/main.js:12]: Nice. We should use a mixin in Blaze Components and Vue as well for this. :-)

**Comment by ManuelDeLeon** [comment]
Looks like normal VM code to me =)

**Comment by mitar** [comment]
Great. Thanks.

**Pull request merged**